src/include/function/comparison/vector_comparison_operations.h

```diff
@@ -164,10 +164,10 @@ class VectorComparisonOperations : public VectorOperations {
         case common::LogicalTypeID::TIMESTAMP: {
             switch (rightTypeID) {
             case common::LogicalTypeID::DATE: {
-                return BinarySelectFunction<common::timestamp_t, common::date_t, FUNC>;
+                return &BinarySelectFunction<common::timestamp_t, common::date_t, FUNC>;
             }
             case common::LogicalTypeID::TIMESTAMP: {
-                return BinarySelectFunction<common::timestamp_t, common::timestamp_t, FUNC>;
+                return &kuzu::function::BinarySelectFunction<common::timestamp_t, common::timestamp_t, FUNC>;
```

Comment:
remove kuzu::function

Reply:
My mistake; something left in from debugging. I'll remove it.